content/en/blogs/install-nfs-server-client-for-kubesphere-cluster.md

```diff
@@ -0,0 +1,347 @@
+   Account: admin
+   Password: P@88w0rd
+   
+   NOTES：
+     1. After you log into the console, please check the
+        monitoring status of service components in
+        "Cluster Management". If any service is not
+        ready, please wait patiently until all components 
+        are up and running.
+     2. Please change the default password after login.
+   
+   #####################################################
+   https://kubesphere.io             20xx-xx-xx xx:xx:xx
+   #####################################################
+   ```
+
+## Verify Installation
+
+You can verify that NFS-client has been successfully installed either from the command line or from the KubeSphere web console.
+
+### Command line
+
+1. Run the following command to check your storage class.
+
+   ```bash
+   kubectl get sc
+   ```
+
+   I did not set `nfs-client` as the default storage class so KubeKey installed OpenEBS for me as well. Expected output:
+
+   ```bash
+   NAME              PROVISIONER                                       RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+   local (default)   openebs.io/local                                  Delete          WaitForFirstConsumer   false                  16m
+   nfs-client        cluster.local/nfs-client-nfs-client-provisioner   Delete          Immediate              true                   16m
+   ```
+
+2. Run the following command to check the statuses of Pods.
+
+   ```bash
+   kubectl get pod -n kube-system
+   ```
+
+   Note that `nfs-client` is installed in the namespace `kube-system`. Expected output:
+
+   ```
+   NAME                                                 READY   STATUS    RESTARTS   AGE
+   calico-kube-controllers-59d85c5c84-ckrpb             1/1     Running   0          16m
+   calico-node-td5cw                                    1/1     Running   0          16m
+   calico-node-v2w2x                                    1/1     Running   0          16m
+   calico-node-xwrd5                                    1/1     Running   0          16m
+   coredns-74d59cc5c6-b764l                             1/1     Running   0          16m
+   coredns-74d59cc5c6-p47m7                             1/1     Running   0          16m
+   kube-apiserver-client1                               1/1     Running   0          16m
+   kube-controller-manager-client1                      1/1     Running   0          16m
+   kube-proxy-2t7n9                                     1/1     Running   0          16m
+   kube-proxy-4wkcr                                     1/1     Running   0          16m
+   kube-proxy-f5f67                                     1/1     Running   0          16m
+   kube-scheduler-client1                               1/1     Running   0          16m
+   metrics-server-5ddd98b7f9-lndw6                      1/1     Running   0          12m
+   nfs-client-nfs-client-provisioner-6fc95f4f79-92lsh   1/1     Running   0          16m
```

Comment:
I think only keeping this line is enough

Reply:
Changed